vm-setup/roles/v1aX_integration_test/tasks/node_reuse.yml

```diff
@@ -161,7 +161,7 @@
 
   - name: Put maxSurge field in KubeadmControlPlane back to default value(1).
```

Comment:
There is no default or original value in the process, the process is stateless.
`Put maxSurge field in KubeadmControlPlane value to 1.`

Reply:
What do you mean? According to `kubeactl explain kubeadmcontrolplane.spec.rolloutStrategy.rollingUpdate` it has default 1.

Reply:
I agree. I am wrong.